jetty9/src/main/resources/loading_pages/default.loading.page.html

```diff
@@ -0,0 +1,188 @@
+		c0.5-0.4,0.8-1,0.8-1.9c0-0.9-0.2-1.5-0.7-1.9c-0.4-0.4-1-0.5-1.8-0.5h-4.7v4.9L674.1,606.3L674.1,606.3z"/>
+</g>
+</svg>
+</div>
+
+<div class="section-by-status" id="status-STARTING">
+  GoCD server is starting. Please wait ...
+</div>
+
+<div class="section-by-status" id="status-STARTED" style="display: none;">
+  GoCD server has started. You will be automatically redirected to it in a few seconds. <a href="#" onclick="window.location.reload(true);">Go directly to the GoCD server.</a>
+</div>
+
+<div class="section-by-status" id="status-UNKNOWN" style="display: none;">
+  Unable to contact the GoCD server. It might not have started. Please check the server logs.
+</div>
+
+<div id="update-check-message"></div>
+</body>
+
+<script type="text/javascript">
+  // https://github.com/yanatan16/nanoajax/blob/cc89609d8f4a17a346b27e2f85c18e14de18a76b/nanoajax.min.js
+  !function(t,e){function n(t){return t&&e.XDomainRequest&&!/MSIE 1/.test(navigator.userAgent)?new XDomainRequest:e.XMLHttpRequest?new XMLHttpRequest:void 0}function o(t,e,n){t[e]=t[e]||n}var r=["responseType","withCredentials","timeout","onprogress"];t.ajax=function(t,a){function s(t,e){return function(){c||(a(void 0===f.status?t:f.status,0===f.status?"Error":f.response||f.responseText||e,f),c=!0)}}var u=t.headers||{},i=t.body,d=t.method||(i?"POST":"GET"),c=!1,f=n(t.cors);f.open(d,t.url,!0);var l=f.onload=s(200);f.onreadystatechange=function(){4===f.readyState&&l()},f.onerror=s(null,"Error"),f.ontimeout=s(null,"Timeout"),f.onabort=s(null,"Abort"),i&&(o(u,"X-Requested-With","XMLHttpRequest"),e.FormData&&i instanceof e.FormData||o(u,"Content-Type","application/x-www-form-urlencoded"));for(var p,m=0,v=r.length;v>m;m++)p=r[m],void 0!==t[p]&&(f[p]=t[p]);for(var p in u)f.setRequestHeader(p,u[p]);return f.send(i),f},e.nanoajax=t}({},function(){return this}());
+</script>
+
+<script type="text/javascript">
+  const intervalSizeInSeconds = 2;
+  const numberOfIntervalsBetweenStatusUpdateCalls = 4;
+
+  const hide = function (element) {
+    element.style = "display: none";
+  };
+
+  const show = function (element) {
+    element.style = "display: inherit";
+  };
+
+  const updateMessage = function (message) {
+    document.getElementById("update-check-message").textContent = message;
+  };
+
+  const handleResponse = function (status) {
+    for (const element of document.getElementsByClassName("section-by-status")) {
+      hide(element);
+    }
+    show(document.getElementById("status-" + status));
+
+  };
+
+  const getStatus = function(callbackFn) {
+    window.nanoajax.ajax({
+      url:     '/go/api/v1/health',
+    }, function (code, responseText) {
+      let status = "UNKNOWN";
+      if (code === 200) {
+        status = "STARTED";
+      } else if (code === 503) {
+        status = "STARTING"
+      }
+
+      callbackFn(status);
+    });
+  };
+
+  const incrementingIndex = (function() {
```

Comment:
please use `var`. I recall @bdpiparva showing that `const` failed on some old browsers. *IE* cough.